### PR TITLE
Cards

### PR DESCRIPTION
**Pull request by krate27**
Preview:
Screenshot (516)
Screenshot (517)

Screenshot (519)

### CODE REVIEW

File: Components/Cards.css [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    width: 350px;
+    height: 200px;
+}
+
+.change-in-bg-width {
+    width: 150px;
+    height: 100px;
+}
+
+.h1yo {
```
Comment (Hamdrive) [inline Components/Cards.css:103]: Consider using a better classname

File: Components/Button.css [hunk 2]
```diff
@@ -0,0 +1,56 @@
+.button-wrapper {
+    display: flex;
+    flex-direction: column;
+    gap: 30px;
```
Comment (aryan-13) [inline Components/Button.css:4]: Relative units like rems are better for responsiveness

File: Docsite-HTML-Files/Card.html [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+      </ul>
+    </aside>
+<div class="h1yo">
+ <h1> Cards </h1>
+
+ <p> Card are used to show user related data collectively, like product details.</p>
+ <p> Our library has six types pf cards to be used.</p>
+ <p>You need to alter html structure as per your need. </p>
+ <p> <strong> Kindly look copy paste from the code snippet as per your requirement and uses. </strong> </p>
+ <iframe
```
Comment (aryan-13) [inline Docsite-HTML-Files/Card.html:74]: The indentation can be improved here

**Review by aryan-13 — COMMENTED** [review]
There are alot of places where absolute units are used, which can be replaced with rems

**Review by Hamdrive — CHANGES_REQUESTED** [review]
Good work done, consider using a formatter to make code more readable
Also the PR mentions only card component but you have also pushed Button as well

**Pull request merged**